test/scenarios/requires-type-checking/tslint.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Do you mean so that we can pass the options to `ts.createProgram()` in the task (line 68)?